wiki/Game_mode/osu!/id.md

```diff
@@ -0,0 +1,291 @@
+
+| Rentang poin kesulitan | Difficulty multiplier |
+| :-: | :-- |
+| 0 - 5 | 2x |
+| 6 - 12 | 3x |
+| 13 - 17 | 4x |
+| 18 - 24 | 5x |
+| 25 - 30 | 6x |
+
+Batas tertinggi point kesulitan adalah 27 poin dengan CS7, OD10 dan HP10. Batas terendah poin kesulitan adalah 2 dengan CS2, OD0 dan HP0.
+
+Biasanya CS tidak bisa di bawah 2 atau di atas 7 (diperlukan modifikasi langsung ke file `.osu`).
+
+Perlu diperhatikan bahwa mod (seperti Hard Rock/Easy) tidak akan mengubah **Difficulty multiplier**.
+
+### Penilaian hit object
+
+**Hit circle:**
+
+- Skor 300, 100 atau 50 akan diberikan dari hit circle tergantung pada seberapa akurat pemain mengkliknya.
+- Sebuah miss diberikan jika pemain tidak mengklik sama sekali pada hit circle atau mengkliknya terlalu awal.
+
+**Slider:**
+
+- Penilaian untuk slider tidak bergantung pada keakuratan ketukan awal.
+- Slider terdiri dari slider tick, yang mencakup titik awal, akhir, dan pengulangan dari slider.
+- Skor 300 diberikan jika pemain berhasil mendapatkan semua slider tick, 100 diberikan jika pemain mendapatkan setidaknya setengah dari mereka, dan 50 diberikan jika pemain berhasil mengumpulkan bahkan salah satu dari tick tersebut.
+- Menekan slider **terlalu awal** tidak akan membuat miss, tetapi akan **menyetel ulang** kombo menjadi **0**.
+  - Tidak mengumpulkan slider tick, yang terlihat ataupun tidak, tidak akan membuat Anda ketinggalan, tetapi akan **mengatur ulang** kombo menjadi **0**.
+  - Gagal menyelesaikan slider sampai akhir tidak akan menyebabkan miss tetapi **tidak** akan menambah kombo.
+- Sebuah miss diberikan jika pemain tidak mengumpulkan slider tick.
+
+**Spinner:**
+
+- Skor 300, 100, atau 50 diberikan tergantung pada jumlah putaran yang dilakukan dibandingkan dengan panjang spinner.
+- Miss diberikan jika pemain tidak dapat memperluas lingkaran spinner ke batas yang telah disesuaikan untuk mendapatkan skor 50
+  - Pada skin v1 : Miss diberikan jika pemain tidak dapat memenuhi meteran yang ada di kedua sisi spinner hingga batas yang telah disesuaikan untuk mendapatkan skor 50.
+
+### Skor/Pengganda Kombo
+
+**berikut adalah hal-hal yang akan menambahkan satu poin ke combo multiplier:**
+
+- Ketukan yang tepat pada hit circle.
+- Ketukan yang berhasil di awal slider.
+- Bola slider yang berhasil mengumpulkan slider tick.
+- Bola slider yang berhasil sampai pada ujung slider atau reverse arrow pada slider.
+- Spinner yang selesai/setengah selesai
+
+**Berikut ini akan mengatur ulang combo multiplier menjadi nol:**
+
+- Miss pada hit circle.
+- Menekan slider terlalu awal.
+- Miss pada awal slider atau slider tick
+- Gagal memperluas lingkaran spinner ke ukuran yang telah disesuaikan.
+  - Pada skin v1 : Gagal mengisi setidaknya setengah dari meteran spinner.
+
+**Hal berikut tidak akan menambah atau mengatur ulang combo multiplier:**
+
+- Miss pada ujung slider (akan menghasilkan Katu atau 100/50 hit)
+- Spinner dengan bonus poin 1,000
+
+## Bilah HP
```

Comment:
should be left as `Health bar` i suppose, literally no one out there has ever referred to the hp bar as `Bilah HP` lol

Reply:
i agree on this one ^